README.md

```diff
@@ -307,6 +307,12 @@ List as many namespaces as you want with `-n` (in this case, `default` and `ingr
 krr simple -n default -n ingress-nginx
 ```
 
+It also supports regex matches that requires permission/RBAC in listing namespaces in target cluster
```

Comment:
Since it will listing namespace, The executor (eg: service account) need to be granted `ClusterRole` related to it. sample:

```yaml
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: list-namespaces
rules:
- apiGroups: [""]
  resources: ["namespaces"]
  verbs: ["list"]
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: list-namespaces-binding
subjects:
- kind: ServiceAccount
  name: krr-account
  namespace: kube-system
roleRef:
  kind: ClusterRole
  name: list-namespaces
  apiGroup: rbac.authorization.k8s.io
```

should this snippet added just like [the others](./tests/single_namespace_permissions.yaml) ?

Reply:
@aantn any other opinion/suggestion for this ?

Reply:
How about

> The `-n` flag also supports regex matches like `-n kube-.*`. To use regexes, you must have permissions to list namespaces in the target cluster.

Reply:
sounds good, updated with text above